### DISCUSSION TITLE
Copilot free tier is not available for a popular open-source repo

### DISCUSSION THREAD

**Discussion by aleksey-hoffman** [community/community#31843] [Copilot Conversations]
![image](https://user-images.githubusercontent.com/61761672/188268533-cbb10a95-04c9-4e5d-8fa1-85281c692bfb.png)

Hey guys, on the [signup page](https://github.com/github-copilot/signup) it says:
> "Copilot is free for maintainers of popular open source software on GitHub"

I'm the maintainer of a quite [popular app](https://github.com/aleksey-hoffman/sigma-file-manager) with 1000+ stars.
The question is how popular your repo has to be to become eligible for the free Github Copilot tier?

**Comment by aleksey-hoffman** [comment]
@github-staff any thoughts, guys?

**Comment by dhrubasaha08** [comment]
@aleksey-hoffman contact github support

**Comment by jfullstackdev** [comment]
Support link, try this but there is no assurance

https://support.github.com/contact/account
